frontend/packages/console-app/src/plugin.tsx

```diff
@@ -136,8 +136,12 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'Dashboards/Overview/Health/Prometheus',
     properties: {
+      // t('console-app~Control Plane (single master)')
       // t('console-app~Control Plane')
-      title: '%console-app~Control Plane%',
+      getTitle: (infrastructure) =>
+        infrastructure?.status?.infrastructureTopology === 'SingleReplica'
```

Comment:
ok, it was bad example. 
there is also this part: 
https://github.com/openshift/origin/pull/25936/files#diff-cb149e746da7ca315f8c9b852dd6f407e04e16a7199587be3ae745fa57fe681eR64

I guess for now just check both, and if both equals to 1, then SNO, or, maybe both equals 1 and master=1?

Reply:
or, maybe, if u write about control plane, so jsut controlTopology = 'SingleReplica' is good enough?

Reply:
@romfreiman added check for both `infrastructureTopology` && `controlPlaneTopology`. If you think we need to check the count of master nodes too, let me know.

Reply:
I think the only check we need is `controlPlaneTopology == 'SingleReplica'`.
The number of workers isn't relevant for the control plane high availability indication.